lib/rendition.js

```diff
@@ -248,6 +242,33 @@ class Rendition {
             console.log("could not determine encoding/charset");
         }
 
+        // return here for pipeline rendition
```

Comment:
The checks below only exists in `asset-compute-sdk` but not in `asset-compute-pipeline`, therefore returning here. 
If we want the checks to also happen in pipeline route, we should create a separate ticket to further investigate if it might cause failure in workers and sdk.